src/university_chatbot/university_conversation_logic_adapter.py

```diff
@@ -38,24 +42,47 @@ def __init__(self, chatbot, **kwargs):
     def can_process(self, statement):
         return True
 
+    def delete_additional_info_after_colon(self, word):
+        index = word.find(':')
+        if index == -1:
+            return word
+        return word[:index]
+
+    def set_to_str_with_colons(self, set):
+        string = ''
+        for elem in set:
+            string += elem + ':'
+        string = string[:-1]
+        return string
```

Comment:
I believe it is better to move such methods for example to `statement_utils` file